core/templates/pages/about-page/about-page.component.html

```diff
@@ -53,7 +53,7 @@ <h2 [innerHTML]="'I18N_ABOUT_PAGE_ABOUT_TAB_HEADING' | translate"></h2>
                   <img [src]="getStaticImageUrl('/general/creator-create-exp.png')" alt="">
                 </picture>
                 <h3 [innerHTML]="'I18N_ABOUT_PAGE_ABOUT_TAB_CREATE' | translate"></h3>
-                <p style="margin-top: -10px;" [innerHTML]="'I18N_ABOUT_PAGE_ABOUT_TAB_CREATE_TEXT' | translate">
+                <p class="oppia-about-tab-text" [innerHTML]="'I18N_ABOUT_PAGE_ABOUT_TAB_CREATE_TEXT' | translate">
```

Comment:
The class name is still too vague. There is a lot of text on the about page. The text exists in tabs, but these three components don't even correspond to separate tabs -- they are all in the same tab.

Please pick a class name which is specific enough to (conceptually) only apply to those components, and not to others.

Reply:
Changed